### ISSUE TITLE
Track Implant ID/Coverslip Type

### ISSUE THREAD

**Issue by mekhlakapoor** [AllenNeuralDynamics/aind-data-schema#250]
**Is your feature request related to a problem? Please describe.**
add a field to craniotomy to track implant id/coverslip type (str in SWR). originally requested by @bjhardcastle in this issue: https://github.com/AllenNeuralDynamics/aind-metadata-service/issues/89

**Describe the solution you'd like**
Add field and then map accordingly in the metadata-service

**Comment by jtyoung84** [comment]
@mekhlakapoor Circle up with Saskia about this

**Comment by mekhlakapoor** [comment]
According to email from Ali, Implant/Coverslip ID are in the NSB database and they added more options

**Comment by saskiad** [comment]
@mekhlakapoor is this different from the `implant_part_number` field?

**Comment by mekhlakapoor** [comment]
uhhh it might be. when was implant_part_number added? The original request was in March 2023 so implant_part_number could've been added after

**Comment by saskiad** [comment]
yeah, I don't remember. It wasn't recently, but I'll see if I can dig it out.

**Comment by saskiad** [comment]
It was already part of the craniotomy schema prior to March 8, 2023 ...

**Comment by saskiad** [comment]
@mekhlakapoor I don't think this is about creating a new field as it looks like we already have the field we need. I think it's that we aren't actually ingesting the implant part number currently, so it should be something to add to the service.
